modules/programs/neomutt.nix

```diff
@@ -3,6 +3,69 @@
 with lib;
 
 let
+  urlencode = lib.replaceChars [
```

Comment:
This probably belongs in `lib.strings`, and maybe it could reuse [lib/ascii-table.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/ascii-table.nix)?

If we keep it in HM we'll probably want to add the file where it's defined to the list of [formatter exceptions](https://github.com/nix-community/home-manager/blob/master/format#L18-L27) since it's really not helping here.

Reply:
I used `lib.escapeURL` as it is now in nixpkgs. :) Thank you!